src/main/webapp/repo-bootstrap/global/blueprints/4000_empty/site/website/index.xml

```diff
@@ -23,7 +23,7 @@
 	<file-name>index.xml</file-name>
 	<internal-name>Home</internal-name>
 	<orderDefault_f>-1</orderDefault_f>
-	<body_html><![CDATA[<h1>Welcome to Your Crafter CMS Site.</h1>
+	<body_html><h1>Welcome to Your Crafter CMS Site.</h1>
```

Comment:
Shouldn't this be xml-escaped?

Reply:
For some reason the IDE's search didn't return all of the ocurrences, went through all the content and removed all CDATA, and escaped properly